q2_feature_table/plugin_setup.py

```diff
@@ -18,7 +18,14 @@
     name='feature-table',
     version=q2_feature_table.__version__,
     website='https://github.com/qiime2/q2-feature-table',
-    package='q2_feature_table'
+    package='q2_feature_table',
+    citation_text=('The Biological Observation Matrix (BIOM) format or: how '
```

Comment:
I think the BIOM format citation makes more sense in q2-types (when we support multiple citations in the framework), because that is where the BIOM format transformers are implemented. This plugin provides general feature table operations that are file format-agnostic.